assembly/onpremises-ide-packaging-war-factory/src/main/java/com/codenvy/onpremises/factory/deploy/FactoryServletModule.java

```diff
@@ -39,6 +41,11 @@ protected void configureServlets() {
         filterRegex(PASS_RESOURCES_REGEXP).through(com.codenvy.onpremises.factory.filter.ReferrerCheckerFilter.class);
 
 
+        Multibinder<FactoryParametersResolver> githubFactoryParametersResolverMultibinder =
```

Comment:
We do not use servlet modules to bind components unless it is servlet/filter or other module that provides servlets/filters